platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/storage/FileSource.java

```diff
@@ -105,7 +109,7 @@ public static synchronized FileSource getInstance(@NonNull Context context) {
    * @return the files directory path
    */
   @NonNull
-  private static String getCachePath(@NonNull Context context) {
```

Comment:
Is this change on purpose?

Reply:
No.  I'll restore it to the previous access.